contrib/completions/bash/buildah

```diff
@@ -690,6 +690,163 @@ return 1
      esac
  }
 
+ _buildah_manifest() {
+     local boolean_options="
+     --help
+     -h
+     --all
+  "
+     subcommands="
+        add
+        annotate
+        create
+        inspect
+        push
+        remove  
```

Comment:
Trailing whitespace - I'm curious why the commit hooks didn't catch this?

Reply:
fixed, thx.